dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/AnnotationBean.java

```diff
@@ -136,19 +137,23 @@ public void destroy() {
     @Override
     public Object postProcessAfterInitialization(Object bean, String beanName)
         throws BeansException {
-        if (!isMatchPackage(bean)) {
+        Class targetClass=bean.getClass();
+        if(AopUtils.isAopProxy(targetClass)){

```

Comment:
Maybe you need to consider isCglibProxy or isJdkDynamicProxy ?

Reply:
That is a different code style.
Maybe format your code first.